Update Indonesian Translations

andr33ww opened the pull request:
- Add New Untranslated XML from value.xml
- Fix some inline untranslated text
- Fix typos
- Better translations for some part (Not all part is better, users will be the judge)

andr33ww committed:
Update Indonesian Translations

- Add New Untranslated XML from value.xml
- Fix some inline untranslated text
- Fix typos
- Better translations for some part (Not all part is better, users will be the judge)

rafaelvcaetano commented:
Thank you for your contribution! Since you have made the changes, I've added some new strings. Do you think you could update your translation to include these new strings? I'm planning on creating a minor release soon, so it would be nice to include this update.

You can find the newly added strings in this commit: https://github.com/rafaelvcaetano/melonDS-android/commit/a9040602cfcaa96f56c425802b07b3b6c827a37e#diff-5e01f7d37a66e4ca03deefc205d8e7008661cdd0284a05aaba1858e6b7bf9103

andr33ww commented:
Thanks too for replying,

Sure thing, wait, let me do do that for a moment.

andr33ww committed:
Update strings.xml

add 3 new lines added in this commit: a904060#diff-5e01f7d37a66e4ca03deefc205d8e7008661cdd0284a05aaba1858e6b7bf9103

andr33ww commented:
done, it's only 3 lines right?

rafaelvcaetano commented:
> done, it's only 3 lines right?

Yes, that's it! Thank you! I'll merge this just before creating the release.

andr33ww commented:
that's a great news, thank you too for letting me in @rafaelvcaetano

rafaelvcaetano commented:
Hello! Sorry for leaving this here for so long. Just letting you know that I'm planning on creating a stable release in the next days. I don't plan on adding any new strings until then (or I'll try to at least). If you want to update your translation, now would be the perfect time.

andr33ww committed:
Update strings.xml

Update the remaining new string.xml

andr33ww committed:
Merge branch 'master' into master

andr33ww commented:
Hi thanks for reaching me out, and sorry I just have the time to update my PR, so here we go, new Indonesian translation based on the new string.xml.
